platforms/pomelo/src/main.rs

```diff
@@ -65,14 +65,21 @@ async fn run_pomelo() {
 
 #[tracing::instrument(name = "Kernel", level = "info")]
 async fn kernel_entry() {
-    let settings = KernelSettings {
-        max_drivers: 16,
-        // TODO(eliza): chosen totally arbitrarily
-        timer_granularity: maitake::time::Duration::from_micros(1),
+    let settings = KernelSettings { max_drivers: 16 };
+
+    let clock = {
+        maitake::time::Clock::new(
+            // TODO(eliza): timer granularity chosen totally arbitrarily
+            Duration::from_micros(1),
+            || {
+                // TODO(anatol) please fix this for me thanks :)
+                unimplemented!("FIGURE OUT HOW TO GET A NORMAL TIMESTAMP OUT OF WASM LOL")
```

Comment:
one way to make this work with the current `Clock` interface is to stuff a `DateTime<Local>` into a global so it can be grabbed by this function. i'll probably just go do that. 

perhaps there's a nicer way to get a global integer timestamp on wasm, but i don't know what it is.